themes/morphosource/views/MyProjects/Media/media_info_html.php

```diff
@@ -398,13 +408,115 @@
 				#$va_mediaFileErrors = $this->getVar("mediaFileErrors");
 	?>
 				<div id="formArea" class="mediaFilesForm"><div class="ltBlueTopRule"><br/>
+
+<div class="jr-group">
```

Comment:
Is this identical to the file upload code snippets used in form_html.php, and are both very modular? If so, could be possible to reduce repetition by moving both code snippets to a single external file and using jQuery.load() to place it in the div? Maybe this isn't a good idea, though, just an idea

Reply:
I thought about sharing the snippets, but since the php behaves differently (single vs. multiple upload forms), I prefer to keep them separate.  I did run into many issues because of the differences and had to make adjustments.